src/test/unit/mocks/opaPublicList.json

```diff
@@ -0,0 +1,113 @@
+                                                                    "reportingRestrictionDetail": [
+                                                                        "Offence Reporting Restriction detail"
+                                                                    ]
+                                                                }
+                                                            ]
+                                                        }
+                                                    },
+                                                    {
+                                                        "partyRole": "DEFENDANT",
+                                                        "individualDetails": {
+                                                            "individualSurname": "IndividualSurname2",
+                                                            "individualFirstName": "individualFirstName2",
+                                                            "offence": [
+                                                                {
+                                                                    "offenceTitle": "Offence title 2",
+                                                                    "offenceSection": "Offence section 2",
+                                                                    "reportingRestrictionDetail": [
+                                                                        "Offence Reporting Restriction detail"
+                                                                    ]
+                                                                }
+                                                            ]
+                                                        }
+                                                    },
+                                                    {
+                                                        "partyRole": "PROSECUTING_AUTHORITY",
+                                                        "friendlyRoleName": "Prosecuting Authority",
+                                                        "organisationDetails": {
+                                                            "organisationName": "organisationName",
+                                                            "organisationAddress": {
+                                                                "line": [
+                                                                    "ORGANISATION ADDRESS"
+                                                                ],
+                                                                "town": "town name",
+                                                                "county": "",
+                                                                "postCode": ""
+                                                            },
+                                                            "offence": [
```

Comment:
You can remove the the offence line from the test file if no offences for PROSECUTING_AUTHORITY